.github/workflows/java_tests.yml

```diff
@@ -162,6 +160,15 @@ jobs:
           service_account_key: ${{ secrets.GCP_SA_KEY }}
           project_id: ${{ secrets.GCP_PROJECT_ID }}
           export_default_credentials: true
+      - name: Set Java Version
+        uses: actions/setup-java@v1
+        with:
+          java-version: 8
+      - name: Remove default github maven configuration
+        # This step is a workaround to avoid a decryption issue of Beam's
+        # gradle-command-action plugin and github's provided maven
+        # settings.xml file
+        run: rm ~/.m2/settings.xml
```

Comment:
Is this a bug in the `gradle-command-action` plugin that we should fix upstream?

Reply:
Thanks Brian!
 
I think the problem is that [setup-java](https://github.com/actions/setup-java) plugin creates by default a `~/.m2/settings.xml `file that cannot be decrypted by `gradle-command-action` plugin because there isn't a `~/.m2/settings-security.xml`. In this context I think is not necessary the `~/.m2/settings.xml` file so we can keep it like this. A possible fix in `gradle-command-action` could be to detect if there is not a `~/.m2/settings-security.xml `file and in that case, do not try to decrypt the password. I saw similar issues [here](https://github.com/mark-vieira/gradle-maven-settings-plugin/issues/15)

Reply:
Ah I see, thank you